### PR TITLE
!feat: added CreateState endpoint to subscription, types update

### PR DESCRIPTION
**Pull request by rostyk-kanafotskyy**
## Type

* ### Feature
  Implements a new feature

## Description

*A brief description of the goals of the pull request.*

## Dependencies

*Other PRs or builds that this PR depends on.*

## Issues

*A list of issues closed by this PR.*

* Fixes #

## Notes

*Any additional notes.*

### CODE REVIEW

File: src/types/subscriptions.d.ts [hunk 1]
```diff
@@ -139,4 +143,6 @@ export interface SubscriptionsEndpoint
   GetAttachedProducts(id: string) : Promise<Resource<SubscriptionOfferingProduct[]>>
 
   GetAttachedPlans(id: string) : Promise<Resource<SubscriptionOfferingPlan[]>>
+
+  CreateState(id: string, action: SubscriptionsStateAction) : Promise<never>
```
Comment (GeorgeBotzakis) [inline src/types/subscriptions.d.ts:147]: What does the new endpoint return?

Comment (rostyk-kanafotskyy) [reply src/types/subscriptions.d.ts:147]: Nothing, actually. I used 'never,' but should I use something else?

Comment (GeorgeBotzakis) [reply src/types/subscriptions.d.ts:147]: @rostyk-kanafotskyy  Not 100% sure, but Promise<void> would have a similar effect?

Comment (rostyk-kanafotskyy) [reply src/types/subscriptions.d.ts]: I did a little thinking about this and changed `Promise<never>` to `Promise<void>`. This is the right option. 
Thanks for pointing this out!

**Review by petro97 — APPROVED** [review]

**Pull request merged**